public/components/marketing-testimonials/9.html

```diff
@@ -0,0 +1,56 @@
+<div class="my-0 mx-auto flex flex-wrap items-center justify-center">
+
+    <!-- Card 1 -->
+    <section class="w-80 rounded-sm shadow-lg bg-slate-900 px-10 py-8 text-white m-2 ">
```

Comment:
Shouldn't be using fixed widths like this, easily removed when switching to `grid` and you wouldn't use `m-2` for spacing with `flex` or `grid` as you can just use `gap-2`.